requirements.txt

```diff
@@ -0,0 +1,13 @@
+-e .
+colour-runner==0.0.4
+coverage==4.0.0
+dj-database-url==0.3
+django==1.8.5
+django-pgcrypto-fields==1.0.0
```

Comment:
This file should list dev/test requirements, whereas project dependencies should be listed in `setup.py`.